setup.py

```diff
@@ -27,7 +27,9 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ],
-    install_requires=['requests>=2.5'],
+    install_requires=[
+        'requests>=2.5',
+    ],
     tests_require=['responses'],
```

Comment:
did you consider my alternate method of specifying test requirements? it makes it so you can easily install test dependencies by running `pip install -e .[test]`

https://github.com/castle/castle-python/pull/22/commits/f044a6bd71cd9c30175f4153413ff45f0cccec4b

Reply:
yes I reverted that because I had troubles to make it working with travis (we can try to add it in the next PR with proper changes to CI config)